bigtable/tests/unit/test_row_data.py

```diff
@@ -439,15 +439,60 @@ def test_state_new_row_w_row(self):
             client._table_data_client.transport.read_rows, request)
 
         yrd._response_iterator = iterator
-        yrd._last_scanned_row_key = ''
         rows = [row for row in yrd]
 
         result = rows[0]
         self.assertEqual(result.row_key, self.ROW_KEY)
+        self.assertEqual(yrd._counter, 1)
 
         yrd._row = object()
         self.assertEqual(yrd.state, yrd.NEW_ROW)
 
+    def test_multiple_chunks(self):
+        from google.cloud.bigtable_v2.gapic import bigtable_client
+
+        chunk1 = _ReadRowsResponseCellChunkPB(
+            row_key=self.ROW_KEY,
+            family_name=self.FAMILY_NAME,
+            qualifier=self.QUALIFIER,
+            timestamp_micros=self.TIMESTAMP_MICROS,
+            value=self.VALUE,
+            commit_row=False,
+        )
+        chunk2 = _ReadRowsResponseCellChunkPB(
+            qualifier=self.QUALIFIER + b'1',
+            timestamp_micros=self.TIMESTAMP_MICROS,
+            value=self.VALUE,
+            commit_row=True,
+        )
+        chunks = [chunk1, chunk2]
+
+        response = _ReadRowsResponseV2(chunks)
+        iterator = _MockCancellableIterator(response)
+        channel = ChannelStub(responses=[iterator])
+        data_api = bigtable_client.BigtableClient(channel=channel)
+        credentials = _make_credentials()
+        client = self._make_client(project='project-id',
+                                   credentials=credentials, admin=True)
+        client._table_data_client = data_api
+        request = object()
+
+        yrd = self._make_one(
+            client._table_data_client.transport.read_rows, request)
+
+        yrd._response_iterator = iterator
+        rows = [row for row in yrd]
+        result = rows[0]
+        self.assertEqual(result.row_key, self.ROW_KEY)
+        self.assertEqual(yrd._counter, 1)
+
+        yrd._row = object()
+        self.assertEqual(yrd.state, yrd.NEW_ROW)
+
+    def test_multiple_chunks(self):
```

Comment:
This should be removed as it is a duplicate from [line 451](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5956/files#diff-e1d4ea33ffcf7ba98a70d7dc9af8ec6eR451)